### ISSUE TITLE
publish link types split from react

### ISSUE THREAD

**Issue by theoephraim** [plaid/react-plaid-link#224]
Would be great if you published the plaid link types themselves separately from the react component, to be used when using plaid link without react.

**Comment by skylarmb** [comment]
hi @theoephraim this is specifically a react package. If you are looking to integrate link in iOS, Android, or React Native, we have other SDKs for that. There are also a number of community maintained SDKs (no official support from Plaid) like `vue-plaid-link`, `ngx-plaid-link` etc.

If none of that solves your use case I would recommend just copying the types you need. The types for initializing link and of the link callbacks are stable, and have not ever been and will not ever be updated in a manor that introduces breaking changes, so you can feel safe copying them.

**Comment by theoephraim** [comment]
I understand. I also did look at the other sdks (vue in my case), but it did not provide the types related to the link js code itself.

I ended up copying the type file and commenting out the _6 lines_ related to the react component rather than Link itself. It just seems a bit silly when most of the file is related to link and could be published separately and used by this component and any others that are not react related. I'm suggesting you publish those types separately and then pull them into this package - which would enable those other SDKs and tools to do the same.
